src/Hng.Application/Features/Languages/Dtos/LanguageResponseDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Hng.Application.Features.Languages.Dtos
+{
+    public class LanguageResponseDto
+    {
+        [JsonPropertyName("language")]
+        public LanguageDto Language { get; set; }
```

Comment:
Please rebane this the data field, so as to follow convention

Reply:
I do not understand what you mean here.